tests/dummy/app/router.js

```diff
@@ -1,12 +1,9 @@
 import EmberRouter from '@ember/routing/router'
-import config from './config/environment'
+import config from 'dummy/config/environment'
 
-const Router = EmberRouter.extend({
-  location: config.locationType,
-  rootURL: config.rootURL
-})
+export default class Router extends EmberRouter {
+  location = config.locationType;
+  rootURL = config.rootURL;
```

Comment:
Looks like there are some semicolons in here that we can remove, I think there's a few in some other files as well.